forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -218,7 +219,10 @@ impl WalletCommands {
 
                 for address in response {
                     let addr = address.0.to_string();
-                    let default_address_mark = if addr == default { "X" } else { "" };
+                    let default_address_mark = default
+                        .as_ref()
+                        .map(|s| if s == &addr { "X" } else { "" })
+                        .unwrap_or("");
```

Comment:
Here we can do `default == Some(addr)`, right?

Reply:
i had that initially, but fmt makes it a bit long to parse, so used this. want me to revert to `if else`?

Reply:
I'm thinking:
```rust
if default.as_ref() == Some(&addr) { "X" } else { "" };
```

Reply:
that's more lean, thx.